src/transport/websocket/mod.rs

```diff
@@ -683,6 +728,11 @@ impl Stream for WebSocketTransport {
             }
         }
 
+        // We have filtered out all `Poll::Ready` events.
+        if should_wake_up {
+            cx.waker().wake_by_ref();
+        }
+
```

Comment:
The same here: we better save and trigger a waker only when adding new elements to `pending(_raw)_connections`.